Library/Formula/argon2.rb

```diff
@@ -0,0 +1,18 @@
+class Argon2 < Formula
+  desc "The password hash Argon2, winner of PHC"
```

Comment:
We might want to shorten this to something like `"Password hash"`. We already know it’s called `argon2` and I’m not sure everybody knows what PHC is.